packages/components/app/styles/components/tabs.scss

```diff
@@ -38,26 +38,18 @@ $indicator-height: 3px;
   overflow-x: auto;
   white-space: nowrap;
   list-style: none;
+  background: linear-gradient(to right, var(--token-color-border-primary), var(--token-color-border-primary)) repeat-x;
+  background-position: bottom floor(calc($indicator-height / 2)) right 0;
+  background-size: 20px 1px;
   -webkit-overflow-scrolling: touch;
-
-  // bottom border
-  &::before {
-    position: absolute;
-    right: 0;
-    bottom: floor(calc($indicator-height / 2));
-    left: 0;
-    z-index: 3;
-    display: block;
-    width: 100%;
-    height: 1px;
-    background: var(--token-color-border-primary);
-    content: "";
-  }
 }
 
 .hds-tabs__tab {
   position: relative; // for positioning indicator
 
+  // prevent overflow gradients from overlaying focused tab
+  &:focus-within { z-index: 2; }
```

Comment:
I _think_ this overlay is still visible (this is in Chrome):
<img width="199" alt="screenshot_1828" src="https://user-images.githubusercontent.com/686239/193014785-d10bc4a8-8112-49f2-9207-a34e151763b1.png">

Reply:
I deleted everything related to overflow for now. Will reexamine later.